WordPress/src/main/java/org/wordpress/android/models/ReaderTag.java

```diff
@@ -176,7 +172,33 @@ public boolean isListTopic() {
      */
     @Override
     public String getLabel() {
-        return getTagDisplayName();
+        if (tagType == ReaderTagType.DEFAULT) {
+            return getTagTitle();
+        } else if (isTagAlphaNumeric(tagDisplayName)) {
```

Comment:
instead of directly accessing `tagDisplayName`, can we use the getter? This should also allow us to remove the parameter in `isTagAlphaNumeric()`.